app/lab/index.template.js

```diff
@@ -13,6 +13,7 @@ const styles = import('./style.js');
 const serverExtensions = [
   import('@jupyterlite/javascript-kernel-extension'),
   import('@jupyterlite/pyolite-kernel-extension'),
+  import('@jupyterlite/robolite-kernel-extension'),
```

Comment:
this isn't necessary, as it's entirely possible to package up a lite kernel as a pip-installable, prebuilt extension... it's a much smaller webpack burden, for sure, which is always good.

Reply:
@bollwyvl Yes! The pip-packages are already hatching here https://github.com/robots-from-jupyter/robotkernel/tree/robolite/lite/jupyterlite-robotkernel/packages and will be merged into master once I'm ready.

You may recall that initially I just grepped everything `pyolite` and copied them to `robolite` to make `robotkernel` work :)

I still need this fork for the hack for Camunda integration, until I learn to make it otherwise: 1) I want to have a custom JS package available for pyolite 2) I need to populate os.environ with a few variables from Camunda. (I could pinpoint those lines ones I have manged to update that branch.)

Reply:
> a custom JS package available for pyolite

right, so your kernel extension will have a `worker.ts`, which is what can actually touch `pyodide`. It is somewhat limiting, as it _doesn't_ get webpacked, it can't really `import` anything, directly, and you'll need to get it the absolute URL of the library you're importing, and in `kernel.ts`, add another [`importScripts`](https://github.com/jupyterlite/jupyterlite/blob/main/packages/pyolite-kernel/src/kernel.ts#L53).

Then, inside `worker.ts`, after everything else gets set up, you'll have access to the [ready `pyodide` and `pyolite`](https://github.com/jupyterlite/jupyterlite/blob/main/packages/pyolite-kernel/src/worker.ts#L68). On pidgy, we happen to only need to hack [python stuff](https://github.com/deathbeds/pidgy/blob/main/lite/jupyterlite-pidgy/src/worker.ts#L66), but having JS works as well.

Note that some of the scoping is a little wonky: if you want somehting to be imported for real, you might have to put it into `user_ns`.

Reply:
For now I resolved my Camunda-needs by following pidgy / robotkernel approach for custom lite package and copy & pasted custom pyolite and robotkernels with required hacks.